Solution for problem 1

ravic499 opened the pull request.

ravic499 committed:
Solution for problem 1

srinathmkce commented:
Good Start !!! Keep Going.

ravic499 commented:
Thanks @srinathmkce , Below is the actual problem statement for this solution,

"If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.

Find the sum of all the multiples of 3 or 5 below 1000."

Kindly review my solution and provide me feedback

diseworks commented on python/problem_1.py:
```diff
@@ -0,0 +1,7 @@
+    for i in xrange(limit):
+        if i % 3 == 0 or i % 5 == 0:
+            result += i
+    print result
```
Please use python 3

ravic499 replied:
Thanks. Fixed it.

diseworks commented on python/problem_1.py:
```diff
@@ -0,0 +1,7 @@
+def calculate(limit):
+    result = 0
+    for i in xrange(limit):
```
please use python 3

diseworks replied:
Try to reduce the complexity of this algorithm

ravic499 replied:
Can you please be more specific, not able to identify the actual issue

diseworks replied:
try to find any other way than using for loop

ravic499 replied:
Sorry, I couldn't find any other way.

diseworks commented on python/problem_1.py:
```diff
@@ -0,0 +1,7 @@
+def calculate(limit):
```
Please add the problem statement

ravic499 replied:
Thanks, Fixed it.

ravic499 committed:
[Project-Euler] Addressed the review comments

Used python 3 specific syntax
Added problem statement

Testing   : Yet to test
Reviewers : diseworks, srinathmkce

nirmal040434 reviewed (commented):
Please provide a meaningful file name.

ravic499 commented:
Thanks @nirmal040434 , I have done the changes

ravic499 committed:
Modified the file name as per review request from Nagaraj

ravic499 reviewed (commented):
Updated the changes
